sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/Infrastructure/MockCheckPointStorage.cs

```diff
@@ -24,18 +24,24 @@ internal sealed class MockCheckPointStorage : PartitionManager
         /// <summary>The primitive for synchronizing access during ownership update.</summary>
         private readonly object _ownershipLock = new object();
 
-        /// <summary>The set of stored ownership.</summary>
-        private readonly Dictionary<(string, string, string, string), PartitionOwnership> _ownership;
-
         /// <summary>The primitive for synchronizing access during checkpoint update.</summary>
         private readonly object _checkpointLock = new object();
 
-        /// <summary>The set of stored checkpoints.</summary>
-        private readonly Dictionary<(string, string, string, string), Checkpoint> _checkpoints;
-
         /// <summary>Logs activities performed by this partition manager.</summary>
         private readonly Action<string> _logger;
 
+        /// <summary>
+        ///   The set of checkpoints held for this instance.
+        /// </summary>
+        ///
+        public Dictionary<(string, string, string, string), Checkpoint> Checkpoints { get; }
```

Comment:
Strange that we hadn't run into a case where we wanted to inject mock data before...

Reply:
Hadn't thought of that, honestly.   I was looking at it as a mock that I wanted to seed data in rather than using the class.   Since you don't seem to have objections, I'm going to leave the change in here to allow for more flexibility with setup.